arch/Kconfig

```diff
@@ -722,6 +722,10 @@ config ARCH_VMA_MAPPING
 	bool "Support runtime memory mapping into SHM area"
 	default n
 
+config ARCH_KVMA_MAPPING
+	bool "Support kernel space virtual mmemory allocation mappings"
```

Comment:
let's remove the prompt, option selected by other option should forbid to be enabled by defconfig

Reply:
Prompt string has been dropped in latest upate. Thanks for teaching this tip.